sdk/python/kfp/compiler/_k8s_helper.py

```diff
@@ -31,13 +31,15 @@ def __init__(self):
 
   def _configure_k8s(self):
     try:
-      config.load_kube_config()
-      logging.info('Found local kubernetes config. Initialized with kube_config.')
-    except:
-      logging.info('Cannot Find local kubernetes config. Trying in-cluster config.')
       config.load_incluster_config()
       logging.info('Initialized with in-cluster config.')
-    
+    except:
+      logging.info('Cannot find in-cluster config, trying the local kubernetes config. ')
+      try:
+        config.load_kube_config()
+        logging.info('Found local kubernetes config. Initialized with kube_config.')
+      except:
+        raise RuntimeError('Forgot to run the gcloud command?')
```

Comment:
Maybe show this link to the user: https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl

Reply:
I like this. Will add it.